File: Back End/C/Chapter 3 - Control Statement/2K221010/patteren_number2_ascending.c
//Loops:

//Pattern Programs

/*

Ex. Number Pattern 2 - Ascending

1
2 2
3 3 3
4 4 4 4
5 5 5 5 5

*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

    for(i=1;i<=count;i++){
        for(j=1;j<=i;j++){
            printf("%d ",i);
        }
        printf("\n");
    }
}

File: Back End/Python/Chapter 05 - Control Statements/2K221229/patteren_number2_ascending.py
# //Loops:
# 
# //Pattern Programs
# 
# /*
# 
# Ex. Number Pattern 2 - Ascending
# 
# 1
# 2 2
# 3 3 3
# 4 4 4 4
# 5 5 5 5 5
# 
# */

count = int(input("Enter the count: "))

for i in range(0, count + 1, 1):
    for j in range(1, i + 1, 1):
        print(i, ' ', end='')
    print(end='\n')